command/sub8_missions/missions/torpedos.py

```diff
@@ -1,20 +1,35 @@
 from txros import util, tf
 import numpy as np
+import rospy
+from sub8_perception.srv import TBDetectionSwitch
+from geometry_msgs.msg import Point, Vector3, Pose, Quaternion
 
+@util.cancellableInlineCallbacks
+def align_to_board(msg, sub):
+    yield sub.move.set_position(target_position).zero_roll_and_pitch().go()
 
 @util.cancellableInlineCallbacks
 def run(sub):
 
     # dive to mission start depth
     print "descending to set course mission depth"
     mission_start_depth = 2.15        # meters
-    sub.to_height(mission_start_depth)
+    sub.to_height(mission_start_depth, 0.5)
```

Comment:
Should the height be negative? I would make a test mission and run it in the sim as to be sure of the sign.
Also it may be good for us to get some logic in all the height functions so that if a positive value gets passed to it, interpret it as a negative instead.